src/ops/execute/execute_test.go

```
@@ -2,6 +2,10 @@ package execute
 
 import (
 	"fmt"
+	"os/exec"
+
+	"github.com/openshift/assisted-installer/src/config"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)